src/models/db/notification.js

```diff
@@ -19,19 +18,9 @@ function NotificationFactory (container, sequelize, Case) {
     }
   }
 
-  PersistentModelMixin(Notification, sequelize, {
-    case_id: Sequelize.UUID,
-    notification_target: Sequelize.TEXT,
-    retry_count: Sequelize.INTEGER,
-    retry_at: Sequelize.DATE
-  }, {
-    indexes: [
-      { fields: ['case_id'] },
-      { fields: ['retry_at'] }
-    ]
-  })
+  Notification.tableName = 'notifications'
```

Comment:
Can't we pass the table name to the ctor? Hardcoding the name in the class is not reuse friendly..